test/services/fileStorage/index.test.js

```diff
@@ -1,5 +1,9 @@
 const mockery = require('mockery');
-const { expect } = require('chai');
+const chai = require('chai');
+chai.use(require('chai-as-promised'));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'chai-as-promised'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4774679)

Reply:
reverted...